### DISCUSSION TITLE
can someone guide me how to do with single footage

### DISCUSSION THREAD

**Discussion by evolakatev** [zju3dv/EasyMocap#294] [Q&A]
i try to follow this guide but i still don't understand how to extract motion capture from single footage

https://chingswy.github.io/easymocap-public-doc/quickstart/quickstart.html
i download demo dataset of the guy playing basketball
i try this command

`python3 apps/demo/mocap.py ${data} --work internet` with the path of my dataset

and the result says

"FileNotFoundError: [WinError 3] The system cannot find the path specified: '0_input\\test\\output-smpl-3d\\smplmesh'"

i don't know how to do next,

**Comment by AzlanCoding** [comment]
ohh for windows right, check the root of the drive.
I had the same problem and it took me soo long to find that all the output files went to `C:/` instead of my current active directory.

**Reply by AzlanCoding** [reply]
Also, (I think you will get this error later) if you get the following traceback when running apps/demo/mocap.py:
```
writing:   0%|                                                                                                                                         | 0/660 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "C:\Users\azlan\EasyMocap\EasyMocap\apps\fit\fit.py", line 38, in <module>
    fitter.fit(body_model, dataset)
  File "c:\users\azlan\easymocap\easymocap\easymocap\multistage\base.py", line 309, in fit
    dataset.write(body_model, body_params, data, camera)
  File "c:\users\azlan\easymocap\easymocap\easymocap\datasets\base.py", line 375, in write
    splitname = splitname[splitname.index(self.annot_args.root)+1:]
ValueError: 'annots' is not in list
```
Add `annname = annname.replace('/','\\')` in line 373 of `easymocap/datasets/base.py`
